erpnext/hr/doctype/employee_referral/employee_referral.json

```diff
@@ -0,0 +1,274 @@
+{
+ "actions": [],
+ "autoname": "format:HR-REF-{####}",
+ "creation": "2021-03-23 14:54:45.047051",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "first_name",
+  "last_name",
+  "full_name",
+  "email",
+  "contact_no",
+  "resume",
+  "column_break_6",
+  "date",
+  "status",
+  "current_employer",
+  "current_job_title",
+  "for_designation",
+  "referrer_details_section",
+  "employee",
+  "employee_name",
+  "column_break_14",
+  "is_applicable_for_referral_compensation",
+  "department",
+  "additional_information_section",
+  "why_this_candidate_is_qualified_for_this_position",
+  "column_break_17",
+  "work_references",
+  "amended_from"
+ ],
+ "fields": [
+  {
+   "fieldname": "first_name",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "First Name ",
+   "reqd": 1
+  },
+  {
+   "fieldname": "last_name",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Last Name",
+   "reqd": 1
+  },
+  {
+   "fieldname": "full_name",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Full Name",
+   "read_only": 1
+  },
+  {
+   "fieldname": "contact_no",
+   "fieldtype": "Data",
+   "label": "Contact No.",
+   "options": "Phone"
+  },
+  {
+   "fieldname": "current_employer",
+   "fieldtype": "Data",
+   "label": "Current Employer "
+  },
+  {
+   "fieldname": "column_break_6",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "date",
+   "fieldtype": "Date",
+   "label": "Date",
+   "reqd": 1
+  },
+  {
+   "allow_on_submit": 1,
+   "fieldname": "status",
+   "fieldtype": "Select",
+   "label": "Status",
+   "no_copy": 1,
+   "options": "Pending\nIn Process\nAccepted\nRejected",
+   "permlevel": 1,
+   "read_only": 1,
```

Comment:
Made it read-only because the status of **Employee referral** should be(always) tracked by the **job applicant** status. if a user wants to cancel a referral, have to cancel a document... no need to create a job applicant in the first place. otherwise, status is synced accordingly. the two-way status syncing is an overhead.

Reply:
But if someone doesn't want to create a Job Applicant, they would want to at least Reject Referral from here itself. In that case, we should check if a job applicant exists against the doc, if not, then show a custom button to change the status.

Reply:
done